mlibrary/lib.umich.edu
# December 2023 dependency updates

# Overview
Monthly dependency update to help maintain lib.umich.edu.

## NPM
Check the [package.json](# Overview
Monthly dependency update to help maintain lib.umich.edu.

## NPM
Check the [package.json](https://github.com/mlibrary/lib.umich.edu/commit/e8c17921b184fd405e081236c96544a14ec9e5bc#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519) file to see all dependency updates.

This update excludes the breaking changes from `rehype-parse`, `rehype-react`, and `unified` that impact the [html component](https://github.com/mlibrary/lib.umich.edu/blob/master/src/components/html.js). 

I have [a ticket](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-103) to circle back and fix the html.component file.

## Testing
- Install the updated packages (`npm install`).
- Start [the site](http://localhost:8000) and check if everything still works (`npm start`).
- Make sure the PR is consistent in these browsers:
) file to see all dependency updates.

This update excludes the breaking changes from `rehype-parse`, `rehype-react`, and `unified` that impact the [html component](https://github.com/mlibrary/lib.umich.edu/blob/master/src/components/html.js). 

I have [a ticket](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-103) to circle back and fix the html.component file.

## Testing
- Install the updated packages (`npm install`).
- Start [the site](http://localhost:8000) and check if everything still works (`npm start`).
- Make sure the PR is consistent in these browsers:
